aries-site/src/components/content/ThemeModeToggle.js

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Button, Tip } from 'grommet';
-import { Moon, Sun } from '@hpe-design/icons-grommet';
+import { Mode } from '@hpe-design/icons-grommet';
 import { useDarkMode } from '../../utils';
 
 export const ThemeModeToggle = ({ active }) => {
   const darkMode = useDarkMode();
   const label = darkMode.value ? 'Switch to Light Mode' : 'Switch to Dark Mode';
-  const icon = darkMode.value ? <Moon /> : <Sun />;
+  const icon = darkMode.value ? <Mode /> : <Mode />;
```

Comment:
Is this the new standard icon for theme mode toggle? Can you point me to where this decision was made and shared with the team?

If this is the new standard, this condition is no longer needed. Also, we should prevent the icon name from being read since the button has a aria-label.
```suggestion
  const icon = <Mode aria-hidden="true" />;
```